Move module into CLI procedure (#2450)
theforeman/foreman-documentation: guides/common/modules/proc_promoting-a-content-view.adoc

[id="cli-promoting-a-content-view_{context}"]
.CLI procedure

> . Promote the Content View using the `hammer content-view version promote` each time:

Reviewer: ```suggestion
* Promote the Content View using Hammer for each lifecycle environment:
```
I think it's better to stick with unnumbered list, because the list items denote alternative steps, not a sequence of steps.

> . Promote the Content View using the `hammer content-view version promote` each time:

Reviewer: ```suggestion
* Promote the Content View using Hammer for each lifecycle environment:
```
I think it's better to stick with unnumbered list, because the list items denote alternative steps, not a sequence of steps.

----
+
Now the database content is available in all environments.
+

> . Alternatively, you can promote Content Views across all lifecycle environments within an organization using the following Bash script:

Reviewer: ```suggestion
* Alternatively, you can promote Content Views across all lifecycle environments within an organization using the following Bash script:
```

> To register a host to your Content View, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering Hosts] in _{ManagingHostsDocTitle}_.

Reviewer 2: Now that it's in between the two sections, this no longer makes sense here. Should move it to after both sections.

+
[source, Bash, options="nowrap" subs="verbatim,quotes"]
----
ORG="_My_Organization_"

> CVV_ID=_My_Content_View_ID_

Reviewer: ```suggestion
CVV_ID=_My_Content_View_version_ID_
```

> . Promote Content Views across all lifecycle environments within an organization using the following Bash script:

Reviewer 2: Being an alternative to the repetitive steps above, it would make more sense to acknowledge the contrast here.
```suggestion
. Or, you can promote Content Views across all lifecycle environments within an organization using the following Bash script:
```

ORG="_My_Organization_"

> CVV_ID=_3_

Reviewer 3: ```suggestion
CVV_ID=_My_Content_View_ID_
```

same below in line 73. Is this a composite content view or content view? I'm asking because of the variable name. Looking at the text, it's a Content View, so maybe rename the variable to `CV`?

Reviewer 2: CVV is content view version. The script uses `hammer content-view version promote`, so I think CVV_ID is correct.

do
   hammer content-view version promote --organization $ORG --to-lifecycle-environment-id $i --id $CVV_ID
done
----

> . Display information about your Content View version to verify that it is promoted to the required lifecycle environments:

Reviewer: ```suggestion
.Verification
* Display information about your Content View version to verify that it is promoted to the required lifecycle environments:
```

+
[options="nowrap" subs="verbatim,quotes"]
----
# hammer content-view version info --id _3_
----
+

> To register a host to your Content View, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering Hosts] in _{ManagingHostsDocTitle}_.

Reviewer: Not sure about the placement of this sentence. Perhaps:
```suggestion
.Next steps
To register a host to your Content View, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering Hosts] in _{ManagingHostsDocTitle}_.
```

----
+
Now the database content is available in all environments.

> +

Reviewer: ```suggestion
```

* Alternatively, you can promote Content Views across all lifecycle environments within an organization using the following Bash script:
+
[source, Bash, options="nowrap" subs="verbatim,quotes"]
----
ORG="_My_Organization_"
CVV_ID=_My_Content_View_Version_ID_

for i in $(hammer --no-headers --csv lifecycle-environment list --organization $ORG | awk -F, {'print $1'} | sort -n)
do
   hammer content-view version promote --organization $ORG --to-lifecycle-environment-id $i --id $CVV_ID
done
----

.Verification
* Display information about your Content View version to verify that it is promoted to the required lifecycle environments:
+
[options="nowrap" subs="verbatim,quotes"]
----

> # hammer content-view version info --id _3_

Reviewer: ```suggestion
# hammer content-view version info --id _My_Content_View_Version_ID_
```

----

.Next steps

> To register a host to your Content View, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering Hosts] in _{ManagingHostsDocTitle}_.

Reviewer: ```suggestion
* To register a host to your Content View, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering Hosts] in _{ManagingHostsDocTitle}_.
```
